src/ILCompiler.ReadyToRun/src/Compiler/DependencyAnalysis/ReadyToRunCodegenNodeFactory.cs

```diff
@@ -452,6 +452,44 @@ protected override IMethodNode CreateMethodEntrypointNode(MethodDesc method)
         protected override IMethodNode CreateUnboxingStubNode(MethodDesc method)
         {
             throw new NotImplementedException();
-        }   
+        }
+
+        protected override ISymbolNode CreateGenericLookupFromDictionaryNode(ReadyToRunGenericHelperKey helperKey)
```

Comment:
These two methods look identical to me. Do we expect the implementations to diverge or could they share implementation with a private method?

Reply:
I must admit I'm not yet sure. My assumption is that at some point they'll need to diverge, otherwise we wouldn't need two different R2R helpers.